Active_Record/.env

```diff
@@ -0,0 +1 @@
+RUBY_ENV=development
```

Comment:
It's fine that this file is in your repository when it's a hometask. But in the real life DO NOT store this file in a repo